### ISSUE TITLE
Parsing error: runtime error: index out of range [8] with length 8

### ISSUE THREAD

**Issue by captain-majid** [icza/screp#17]
Hi there, I face this error when parsing this [replay](https://drive.google.com/open?id=1pzlq2YCb36pBJnbH2HQSZfQyxJ0GHsJs), using v1.1.2:
```
2019/10/16 08:29:01 Parsing error: runtime error: index out of range [8] with length 8
2019/10/16 08:29:01 Stack: goroutine 1 [running]:
github.com/icza/screp/repparser.parseProtected.func1(0x12463eb8)
        /home/icza/gows/src/github.com/icza/screp/repparser/repparser.go:114 +0xbe
panic(0x5030e0, 0x125fe4a0)
        /usr/local/go/src/runtime/panic.go:679 +0x162
github.com/icza/screp/repparser.parseMapData(0x12638000, 0x374e4, 0x374e4, 0x124041a0, 0x374e4, 0x0)
        /home/icza/gows/src/github.com/icza/screp/repparser/repparser.go:629 +0x846
github.com/icza/screp/repparser.parse(0x535cd0, 0x12450120, 0x12420101, 0x12420bb8, 0x535410, 0x124061e8)
        /home/icza/gows/src/github.com/icza/screp/repparser/repparser.go:199 +0x17f
github.com/icza/screp/repparser.parseProtected(0x535cd0, 0x12450120, 0x530101, 0x0, 0x0, 0x0)
        /home/icza/gows/src/github.com/icza/screp/repparser/repparser.go:120 +0x81
github.com/icza/screp/repparser.ParseFileSections(0x12412190, 0x4a, 0x101, 0x0, 0x0, 0x0)
        /home/icza/gows/src/github.com/icza/screp/repparser/repparser.go:87 +0xb4
github.com/icza/screp/repparser.ParseFile(...)
        /home/icza/gows/src/github.com/icza/screp/repparser/repparser.go:74
main.main()
        /home/icza/gows/src/github.com/icza/screp/cmd/screp/screp.go:55 +0xae
Failed to parse replay: parsing
```

**Comment by icza** [comment]
Thanks for reporting it, will check it.

**Comment by captain-majid** [comment]
BTW is there is any way to convert scr replays to 1.16 format ? I would love to analyze those replays using CoachAI, as Blizzard's support to BWAPI might take forever.

**Comment by icza** [comment]
Theoretically it's possible, but I don't know of any tools that does this.

**Comment by icza** [comment]
Fixed the issue, released v1.1.3 which now properly parses these replays.

https://github.com/icza/screp/releases/tag/v1.1.3

Thanks again for reporting it.
